templates/js/api-plugin-from-scratch-oauth/README.md

```diff
@@ -0,0 +1,73 @@
+# Overview of the Copilot Plugin template
+
+## Build a Copilot Plugin from a new API with Azure Functions
+
+With Copilot extensibility, you can augment Copilot for Microsoft 365 with custom skills and organizational knowledge specific to your enterprise and users to enable truly spectacular AI scenarios. For example:
```

Comment:
I suggest updating the title to include authentication mechanism here: `Build a Copilot Plugin from a new API authenticated by OAuth with Azure Function`